README.md

```diff
@@ -0,0 +1,125 @@
+              longitude: 4.01728, //defined by user map pin positionning
+              keyword: 47, //second level keyword id
+              author_id: 1,
+              global_grade: 4, //default
+              accessibility: 5, //user input
+              condition: 3, // user input
+              functional: 4, //user input
+            }
+  ```
+
+### Post '/picture' :
+
+  This aims to get the picture of the newly created POI., 
+  We use _formidable_ library to "transform" the picture date in json while storing the picture itself it in a folder with its Id (public/image/here).
+  The link between the POI and the new image is done by storing the picture ID on creation, and reusing it inside the POI creation to reach the storage folder.
+
+data sent from front when submitting file looks like :
+![](https://i.imgur.com/C3oiE56.png)
+
+data transformed by formidable looks like :
+![](https://i.imgur.com/oK1JFai.png)
+
+then the fs.rename sends the picture in the dedicated folder with its initial name and extension, and the following query pushes its name in the database so that it can be used in front to call a url to the image to display.
+
+
+## Get
+
+### Get '/sample/:latitude/:longitude'
+ 
+ it allows get data from the database to display a first sample of two POI per theme (first level keywords) around the geolocated user when the application starts.
+ the radius of the display zone is currently set at 3km/3000m in getSamplePoisInfosCriteria.js
+
+ example : 'sample/49.2653016/4.0140353' will get :
+
+ ``` javascript
+ {
+        "id": 23,
+        "name": "Table de ping-pong - Charles Arnould",
+        "author": "Wilder",
+        "picture_url": "Default_Picture.webp",
+        "firstImportanceKeyword": "Sport & Loisirs",
+        "creation_date": "2019-06-17T22:00:00.000Z",
+        "localisation": [
+            49.2646,
+            4.01292
+        ],
+        "grades": {
+            "average": 5,
+            "accessibility": 5,
+            "condition": 5,
+            "functional": 5
+        },
+        "keywordId": 11
+    },
+    {
+        "id": 19,
+        "name": "Table de ping-pong - Quartier Libre",
+        "author": "Wilder",
+        "picture_url": "Default_Picture.webp",
+        "firstImportanceKeyword": "Sport & Loisirs",
+        "creation_date": "2019-06-17T22:00:00.000Z",
+        "localisation": [
+            49.257,
+            4.01972
+        ],
+        "grades": {
+            "average": 2,
+            "accessibility": 1,
+            "condition": 2,
+            "functional": 3
+        },
+        "keywordId": 11
+    },
+...
+]
+```
+
+
+### Get '/keyword'
+
+it gets all the keywords data registered inside the database.
+
+### Get '/filter/:keyword'
+
+it allows to get data of every POI linked with a specified second level keyword (category).
+
+
+### Get 'filterKeyword1/:keyword'
+
+it allows to get date of every POI linked with a specified first level keyword (theme)
+
```

Comment:
Give an example of JSON in all the routes

Reply:
done